### ISSUE TITLE
Sites for follow up with habitat confirmation

### ISSUE THREAD

**Issue by NewGraphEnvironment** [NewGraphEnvironment/fish_passage_fraser_2023_reporting#23]
Place to keep track of sites that appear to have valuable habitat and significant passage issues (especially outlet drops).  It is nice to have sites that are on small roads and unpaved roads too where actions are easier to achieve in the short term.  Looking through photos - some of these will have the site number circled. The photos on onedrive could be actually really useful for that....

- [ ] 5400202 - Tributary to Francois Lake (need to confirm lake name  - Lakes layer is missing)

**Comment by lucy-schick** [comment]
Sites to follow up with from the simpcw:

- [ ] 13905537 and 13900028 - Trib to Fraser
    - 13905537 is a railway crossing just upstream of 13900028 on penny street, which is the only access to the town of Penny. Crossing 13900028 is at a high risk of washout is crossing 13905537 gets lots of flow. 
    - 13905537 has a big concrete outlet drop and 2 of the 3 culverts are blocked by vegetation.
